gradle-plugin/src/main/java/org/lsposed/lsparanoid/plugin/LSParanoidExtension.kt

```diff
@@ -21,7 +21,9 @@ import com.android.build.api.variant.Variant
 
 open class LSParanoidExtension {
     var seed: Int? = null
+    @Deprecated("Use 'classFilter: (String) -> Boolean'.")
```

Comment:
remove it and bump ver

Reply:
> remove it and bump ver

Should I remove the `@Deprecated` annotation or the `global` property?

Reply:
both